packages/flutter_tools/lib/src/build_system/targets/web.dart

```diff
@@ -190,13 +193,16 @@ class Dart2JSTarget extends Target {
   Future<void> build(Environment environment) async {
     final BuildMode buildMode = getBuildModeForName(environment.defines[kBuildMode]);
     final bool sourceMapsEnabled = environment.defines[kSourceMapsEnabled] == 'true';
+    final bool nativeNullAssertions = environment.defines[kNativeNullAssertions] == 'true';
```

Comment:
Should these have `MissingDefineException` checks?

Reply:
I think its okay to assume a default

Reply:
Works now, could catch something in a refactor in the future.  Up to you though, you are the `assemble` Architect.

![](https://miro.medium.com/max/1680/1*OlY_7egaXZYKJu3QZYmANA.jpeg)